willow/app/renderers/nested_identifier_attribute_renderer.rb

```diff
@@ -0,0 +1,24 @@
+class NestedIdentifierAttributeRenderer < Hyrax::Renderers::AttributeRenderer
+  private
+  def attribute_value_to_html(value)
```

Comment:
could this be moved to an `.erb` template ?

Reply:
Similarly with the renderers, Hyrax use a .rb file and place them all in app/renderers. We can leave it this way for now, as we are doing it the Hyrax way.